webcompat/static/js/lib/wizard/helpers/postMessageParser.js

```diff
@@ -5,17 +5,16 @@
 import notify from "../notify.js";
 import { isSelfReport, convertToDataURI } from "../utils.js";
 import { blobOrFileTypeValid, isImageDataURIValid } from "../validation.js";
+import { sendAnalytics } from "../analytics.js";
 
 const updateStep = (id, data) => notify.publish("updateStep", { id, data });
 const updateUrl = (url) => updateStep("url", { url });
 const updateHidden = (data) => updateStep("hidden", { data });
 const updateScreenshot = (dataURI) => updateStep("screenshot", { dataURI });
 
 const setAnalytics = (campaign, source) => {
-  if (!window.setAnalyticsData) return;
-
   if (campaign && source) {
```

Comment:
Noticed during testing that here `setAnalyticsData` is relying on a function that was available globally. Since it's no longer possible with the new bundles, I've changed it and moved to a separate module.